backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxSparkPlanExecApi.scala

```diff
@@ -56,6 +54,16 @@ class VeloxSparkPlanExecApi extends ISparkPlanExecApi {
   override def genRowToArrowColumnarExec(child: SparkPlan): RowToArrowColumnarExec =
     new VeloxRowToArrowColumnarExec(child)
 
+  /**
+   * Generate FilterExecTransformer.
+   *
+   * @param condition: the filter condition
+   * @param child: the chid of FilterExec
+   * @return the transformer of FilterExec
+   */
+  override def genFilterExecTransformer(condition: Expression, child: SparkPlan)
+    : FilterExecBaseTransformer = VeloxFilterExecTransformer(condition, child)
```

Comment:
@marin-ma How about using sth like "if (isArrowBackend)" to create the orignal filter transformer for Arrow?